src/types/FateInt.js

```diff
@@ -5,7 +5,14 @@ class FateInt extends FateData {
   constructor(value) {
     super('int')
 
-    this._value = BigInt(value)
+    try {
+      this._value = BigInt(value)
+    } catch (error) {
+      if (/^Cannot convert .* to a BigInt$/.test(error.message)) {
```

Comment:
Using calldata the developer may not use BigInts explicitly, so it may be unclear why an argument should be convertible to BigInt. "number" is more general than "BigInt" so the developer would guess that the problem is rather on his side than in the internals of calldata.

But I'm not 100% sure about it and ok to revert this if @marc0olo @mradkov agrees that this is not needed.

Reply:
Mhhhh, when thinking about this I don't think we really need this. I mean every experienced dev should know how to handle this or whats the problem, right? Personally I favor the original error message.

Reply:
That reason, don't hold for me. Please revert.